crates/bifrost/src/providers/replicated_loglet/sequencer/mod.rs

```diff
@@ -35,6 +35,45 @@ use super::{
 use crate::loglet::{util::TailOffsetWatch, LogletCommit, OperationError};
 use appender::SequencerAppender;
 
+pub(crate) const RECORDS_ENQUEUED_COUNT: &str = "restate.sequencer.records_enqueued.total";
+pub(crate) const RECORDS_COMMITTED_COUNT: &str = "restate.sequencer.records_committed.total";
+pub(crate) const RECORDS_ENQUEUED_BYTES: &str = "restate.sequencer.records_enqueued.bytes";
+pub(crate) const RECORDS_COMMITTED_BYTES: &str = "restate.sequencer.records_committed.bytes";
+pub(crate) const LOG_SERVER_LATENCY: &str = "restate.sequencer.log_server_latency.ms";
```

Comment:
Can we namespace those under bifrost to be consistent with the rest?
For instance: `restate.bifrost.replicatedloglet.sequencer.enqueue.total"

Also, those should live in `replicated_loglet/metric_definitions.rs`

One last note, s/latency/duration/ please for consistency with prometheus standards.